core/storage/suggestion/gae_models.py

```diff
@@ -2289,34 +2289,45 @@ def fetch_page(
 
         sorted_results: List[
             TranslationSubmitterTotalContributionStatsModel] = []
+        filtered_results: List[
+            TranslationSubmitterTotalContributionStatsModel] = []
         today = datetime.date.today()
+        more: bool
+        next_result_model: Sequence[
+            TranslationSubmitterTotalContributionStatsModel] = []
+        check_offset: int
 
         if num_days_since_last_activity is not None:
```

Comment:
The first sort property must be the same as the property to which an inequality filter is applied. Thus, the inequality filter on last_activity can not be used here and we have implemented it separately below. Learn more about this here:
https://cloud.google.com/appengine/docs/legacy/standard/go111/datastore/query-restrictions#properties_used_in_inequality_filters_must_be_sorted_first.

Reply:
Ah, right. Forgot about that quirk. Can you please add a comment explaining this with a link to the documentation for future readers?

Reply:
I have added it above on line 2273 where we are actually defining the sort query. Sean suggested that it would be better to reference it while defining the sort query itself rather than later.